### PR TITLE
Add overlay for F1

### PR DESCRIPTION
**Pull request by DiUst**

### CODE REVIEW

File: Umidigi/F1/res/values/bools.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_hotswapCapable">true</bool>
+    <bool name="config_device_volte_available">false</bool>
```
Comment (phhusson) [inline Umidigi/F1/res/values/bools.xml:4]: why?

File: Umidigi/F1/res/values/bools.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_hotswapCapable">true</bool>
+    <bool name="config_device_volte_available">false</bool>
+    <bool name="config_device_vt_available">false</bool>
```
Comment (phhusson) [inline Umidigi/F1/res/values/bools.xml:5]: why?

File: Umidigi/F1/res/values/bools.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_hotswapCapable">true</bool>
+    <bool name="config_device_volte_available">false</bool>
+    <bool name="config_device_vt_available">false</bool>
+    <bool name="config_device_wfc_ims_available">false</bool>
```
Comment (phhusson) [inline Umidigi/F1/res/values/bools.xml:6]: why?

**Comment by DiUst** [comment]
These are stock files. I used the instructions. But I just need the status bar to display correctly. Now I understand that this needs dimens.xml  to be edited, I will close the extraction request, I need to redo it.

**Pull request closed without merging**